src/reducers/local-storage.js

```diff
@@ -0,0 +1,21 @@
+export const getReduxState = () => {
+  try {
+    const registeredState = localStorage.getItem('state');
+    if (registeredState === null) {
+      return undefined;
+    }
+    return JSON.parse(registeredState);
+  } catch (error) {
+    return undefined;
+  }
+};
+
+export const saveReduxState = (currentState) => {
```

Comment:
Same here. I think you should avoid using implementation details in your function names. If we decide to swap out our state management library for something like MobX, we won't need to rename these functions.